### PR TITLE
llvm-libs: new package

### PR DESCRIPTION
**Pull request by lb90**
Make a dedicated package with the LLVM runtime libraries.

Inspired by https://archlinux.org/packages/extra/x86_64/llvm-libs/

### CODE REVIEW

File: mingw-w64-clang/PKGBUILD [hunk 1]
```diff
@@ -390,6 +392,19 @@ package_llvm() {
   sed -e "s|${PREFIX_WIN}|${MINGW_PREFIX}|g" -i ${pkgdir}/${MINGW_PREFIX}/lib/cmake/llvm/LLVMExports.cmake
 }
 
+package_llvm_libs() {
```
[the remark this answers is not in the document]
Comment (lb90) [reply mingw-w64-clang/PKGBUILD]: THank you, now fixed!

**Comment by lb90** [comment]
> OSL is not compatible/buildable with LLVM-16.

Indeed, dropped the commit.

**Comment by lb90** [comment]
Hi! Any updated on this?

Let me know if anything needs to be changed :)

**Comment by lazka** [comment]
The following packages currently link against those DLLs, so there is likely potential to clean up runtime dependencies there:

```
mingw-w64-spirv-llvm-translator
mingw-w64-include-what-you-use
mingw-w64-mesa
mingw-w64-creduce
mingw-w64-qt6-tools
mingw-w64-lldb
```

either way, this looks good, thanks!

**Comment by lazka** [comment]
Should we make clang depend on llvm-libs instead of llvm now too? Like arch does: https://gitlab.archlinux.org/archlinux/packaging/packages/clang/-/blob/5be811a3ca921c82b47d1f5f20c544bca7029a4e/PKGBUILD#L15

**Comment by lazka** [comment]
Should we make clang and lld now depend on llvm-libs instead of llvm?

**Pull request merged**